javaharness/java/arcs/android/ArcsService.java

```diff
@@ -84,7 +84,8 @@ public void registerRenderer(String modality, IRemoteOutputCallback callback) {
 
       @Override
       public boolean addManifests(List<String> manifests) {
-        return arcsShellApi.addManifests(manifests);
+        // TODO: Implement once dynamic manifests are needed.
+        return true;
```

Comment:
Could we do this instead? `throw new UnsupportedOperationException();`

That's way less surprising.

Reply:
Done.